src/DonationForms/resources/registrars/templates/fields/Amount/CustomAmount.tsx

```diff
@@ -42,15 +43,17 @@ const CustomAmount = (
                      */
                     groupSeparator.replace(/\u00A0/g, ' ')
                 }
-                className="givewp-fields-amount__input givewp-fields-amount__input-custom"
+                className={`${styles.customAmount} givewp-fields-amount__input givewp-fields-amount__input-custom`}
                 aria-invalid={fieldError ? 'true' : 'false'}
                 id="amount-custom"
                 name="amount-custom"
                 placeholder={__('Enter custom amount', 'give')}
                 defaultValue={defaultValue}
                 value={value}
                 decimalsLimit={2}
-                onValueChange={(value) => {onValueChange(value !== undefined ? value : '')}}
+                onValueChange={(value) => {
+                    onValueChange(value !== undefined ? value : '');
+                }}
```

Comment:
Let's add an unreleased tag for these changes.

Reply:
Thanks! Resolved: 9517e413377d8fad3f20f433cafcc6181aa0005d